tests/jballerina-unit-test/src/test/resources/test-src/bala/test_bala/object/test_bir_negative.bal

```diff
@@ -0,0 +1,20 @@
+import bir/objs;
+
+function f1(objs:Bar bar) {
+    assertEquality(bar is objs:Foo, false);
+}
+
+function f2(objs:Foo foo) {
+    assertEquality(foo is objs:Bar, false);
+}
+
+function f3(objs:Xyz xyz) {
+    assertEquality(xyz is objs:Qux, false);
+}
+
+function f4(objs:Qux qux) {
+    assertEquality(qux is objs:Xyz, false);
+}
+
```

Comment:
Add another test that checks whether its of the same type. Although the actual test would not run, there wont be a compile time. Furthermore, can you add few positive tests as well that validates the types at runtime. Assigning it a any and validating later. Make sure add for both isolated/distinct together and seperately
eg:
```
function f5(objs:Qux qux) {
    assertEquality(qux is objs:Qux, true);
}
```

Reply:
Added.